pages/furtherreading.vue

```diff
@@ -0,0 +1,132 @@
+          author: "Lizanne Henderson (Basingloke: Palgrave Macmillan, 2016)",
+          link:"https://archive.org/details/witchcraftfolkbe0000hend",
+          name: "Link to this work on the Internet Archive website"
+        },
+        {
+          title: "Enemies of God: The Witch-Hunt in Scotland",
+          author: "Christina Larner (London: Chatto & Windus, 1981)",
+          link:"https://archive.org/details/enemiesofgodwitc00larn_0",
+          name: "Link to this work on the Internet Archive website"
+        },
+        {
+          title: "Witch-Hunting in Scotland: Law, Politics and Religion",
+          author: "Brian P. Levack (London: Routledge, 2008)",
+          link:"https://www.taylorfrancis.com/books/mono/10.4324/9780203089507/witch-hunting-scotland-brian-levack",
+          name: "Link to this work on the Taylor Francis webiste"
+
+        },
+        {
+          title: "The Witches of Fife: Witch-Hunting in a Scottish Shire",
+          author: "Stuart Macdonald (East Linton: Tuckwell, 2001)",
+          link:"https://archive.org/details/witchesoffifewit0000macd",
+          name: "Link to this work on the Internet Archive website"
+        },
+        {
+          title: "The Visions of Isobel Gowdie: Magic, Witchcraft and Dark Shamanism in Seventeenth-Century Scotland",
+          author: "Emma Wilby (Brighton: Sussex Academic Press, 2010)",
+          link:"https://archive.org/details/emma-wilby-the-visions-of-isobel-gowdie-magic-witchcraft-and-dark-shamanism-in-s",
+          name: "Link to this work on the Internet Archive website"
+        },
+        {
+          title: "Witches of the North: Scotland and Finmark",
+          author: "Liv Helene Willumson (Leiden: Brill, 2013)",
+          link:"https://brill.com/display/title/24241",
+          name: "Link to this work on the BRILL website"
+        },
+        {
+          title: "Witchcraft in Early Modern Scotland: James VI's Demonology and the North Berwick Witches ",
+          author: "Lawrence Normand and Gareth Roberts  (Liverpool: Liverpool University Press, 2000)",
+          link:"https://www.jstor.org/stable/j.ctt5vjmvw",
+          name: "Link to this work on the JSTOR website"
+        },
+        {
+          title: "Witch Hunt Podcast",
+          author: "Presented by Louise Yeoman and Susan Morrison (BBC Scotland, 2019)",
+          link:"https://www.bbc.co.uk/programmes/p07rn38z/episodes/player",
+          name: "Link to this work on the BBC website"
+        },
+        {
+          title: "Lucy Worsley Investigates: The Witch Hunts, ",
+          author: "Presented by Lucy Worsely (BBC, 2022)",
+          link:"https://www.bbc.co.uk/programmes/p0bvhk3w",
+          name: "Link to this work on the BBC website"
+        },
+      ]
+
+   }),
+   methods: {
+
+   },
+   computed : {
+
+   },
+   mounted: function() {
+   },
+   layout: 'basic'
+ };
+</script>
+
+<style scoped>
+.custom-margin {
```

Comment:
Should we be using tailwind instead of raw CSS here? Just worried we are mixing techniques.